protos/protos/public/modeldb/DatasetService.proto

```diff
@@ -16,6 +16,10 @@ message Dataset {
     string id = 1;//For backend reference
     string name = 2;
     string owner = 3;
+    oneof owner_tracking {
+        uint64 owner_id = 35;
+        common.GroupId group_owner_id = 36;
+    }
```

Comment:
```suggestion
    oneof owner_tracking {
        uint64 owner_id = 17;
        common.GroupId group_owner_id = 18;
    }
```

Reply:
why do we need it?

Reply:
If we make sequence number different for the same variable names it will be harder to support changes for them

Reply:
I didn't know this would be an issue. I know that we never want to change a sequence number once there is ever a field that uses it. But I did not think that you need to keep them the same across different protos. 🤔 My initial reaction would have been the same as Anand's.

Reply:
I don't think that it is a big deal. But if you both think that way, no problem, I'll change it

Reply:
updated